ever_given/ever_given/run.py

```diff
@@ -23,29 +23,43 @@ def main():
         default="",
     )
     _parsed, unknown = parser.parse_known_args()
-    # print("unknown", unknown)
+
     if len(unknown) % 2 != 0:
         raise ValueError("Must pass key/value pairs")
     for idx in range(len(unknown) // 2):
         arg = unknown[idx * 2]
-        parser.add_argument(arg)
+        orig_arg_without_dashes = arg[2:]
+        parser.add_argument(arg, metavar=orig_arg_without_dashes)
     args = parser.parse_args()
+    # Hacking into argparse internals to map the argument that may have
+    # dashes to argparse's output
+    ag = parser._optionals
+    group_actions = ag._group_actions
+    dashed_by_key = {
+        group_action.dest: group_action.metavar for group_action in group_actions
+    }
+
     kwargs = {}
     file_kwargs = {}
-    FILE_PREFIX = "file_"
+    FILE_PREFIX = "file-"
     for key, value in vars(args).items():
+        new_key = dashed_by_key.get(key)
+        if new_key is not None:
+            key = new_key
         if key.startswith(FILE_PREFIX):
             file_kwargs[key[len(FILE_PREFIX) :]] = value
         else:
             if key not in ("command", "container_uri", "output_keys"):
                 kwargs[key] = value
+    print("container", args.container_uri)
     print("command?", args.command)
     print("file kwargs", file_kwargs)
     print("kwargs", kwargs)
     output_file_keys = args.output_keys.split(",")
     output_dir = "evergiven_output"
     print("Putting output into", output_dir)
     results = {}
+    return
```

Comment:
@robbason I think we forgot to remove the `return` we used for debugging 😝

Reply:
whoops, fixed!